### ISSUE TITLE
Feature Request | Set keras random state with tensorflow

### ISSUE THREAD

**Issue by jolespin** [tensorflow/tensorflow#14669]
Is it possible to set the `random_seed` in `keras` with `tensorflow`?  Would you need to set it to a specific graph? I have not found out how to set the random state in `keras` and if this doesn't exist could this be a feature in future versions?

**Comment by facaiy** [comment]
The feature request has been discussed for half and one year in fchollet/keras#2280, however there seems nothing new. I believe @fchollet must care the issue.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by jolespin** [comment]
How should I update this? I'm not sure how to implement a random_state in keras that transfers to tensorflow.

**Comment by facaiy** [comment]
Hi, @jolespin. Let's wait for reply from tensorflower.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activityand the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 139 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 46 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 61 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 76 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 91 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 106 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 128 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
